containers/push.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: localhost
+  vars:
+    container_cli: 'docker'
+  vars_files:
+    - vars/defaults.yaml
+  tasks:
+    - name: 'Check for podman'
```

Comment:
given both playbooks have "check for podman" and "set_fact" steps, should this be moved to a "select_container_cli" role?